modules/nodes-nodes-configuring-graceful-shutdown.adoc

```diff
@@ -0,0 +1,187 @@
+// Module included in the following assembly:
+// * nodes/nodes-nodes-graceful-shutdown
+
+:_content-type: PROCEDURE
+[id="nodes-nodes-activating-graceful-shutdown_{context}"]
+= Configuring Graceful Node Studown
+
+To configure Graceful Node Shutdown (GNS) for a node, create a `KubeletConfig` custom resource (CR) to specify a grace period and shutdown profile for specific nodes. If GNS  is not configured, the default grace period is `None`.
+
+
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
```

Comment:
I don't think we want to include this procedure in the documentation as it's a demo of how the feature works, not instructions about how to use it.

Reply:
Replaced.